otherlibs/unix/unixLabels.mli

```diff
@@ -580,15 +580,15 @@ val map_file :
   copy-on-write of the modified pages; the underlying file is not
   affected.
 
-  [Genarray.map_file] is much more efficient than reading
+  {!Unix.map_file} is much more efficient than reading
```

Comment:
```suggestion
  {!map_file} is much more efficient than reading
```

Reply:
I was not sure, thank you for the suggestion.